app/layout.tsx

```diff
@@ -95,13 +95,27 @@ export default async function RootLayout({
                     <Link
                       href="https://www.npmjs.com/package/ms"
                       target="_blank"
+                      rel="noopener noreferrer"
                       className="link"
                     >
                       ms
                     </Link>{" "}
                     string e.g. <span className="font-bold">1 day</span> or{" "}
                     <span className="font-bold">30 minutes</span>
                   </li>
+                  <li className="list-item">
```

Comment:
I think this and the above should be in a sub list that says `one of timeframe (default) or cron may be provided`